guides/doc-Upgrading_and_Updating/topics/migrating_linux_7_to_linux_8.adoc

```diff
@@ -0,0 +1,44 @@
+= Migrating from RHEL 7 to RHEL 8
+
+{ProjectName} is supported on both Red Hat Enterprise Linux (RHEL) 7 and RHEL 8.
+
+Subsequent minor versions of {Project} might be supported on RHEL 8 only.
+
+If you have {ProjectName} running on RHEL 7, you must migrate to RHEL 8 to stay up-to-date on minor releases.
+
+There are two ways to migrate your server to RHEL 8:
+
+//.Unordered list
+
+* Create a backup of your server on RHEL 7, perform a fresh install of a minimal RHEL 8 instance, and restore the backup on the RHEL 8 instance, or
+* Perform an in-place migration on your server using the Leapp utility.
+
+[discrete]
+=== Fresh Install Method
```

Comment:
Making a note that we will need a provision in this section for users with external databases.